main.tf

```diff
@@ -56,6 +56,10 @@ provider "helm" {
   }
 }
 
+provider "kubernetes" {
```

Comment:
I'm sorry I stepped on this, this'll need a rebase to avoid double counting: https://github.com/chainguard-images/images/blob/main/main.tf#L53

Reply:
Cool cool, thanks a lot 🥺